build.sbt

```diff
@@ -407,7 +407,9 @@ lazy val codegen = projectMatrix
       "smithyOrg" -> Dependencies.Smithy.org,
       "smithyVersion" -> Dependencies.Smithy.smithyVersion,
       "alloyOrg" -> Dependencies.Alloy.org,
-      "alloyVersion" -> Dependencies.Alloy.alloyVersion
+      "alloyVersion" -> Dependencies.Alloy.alloyVersion,
+      "smithy4sOrg" -> "com.disneystreaming.smithy4s",
```

Comment:
```suggestion
      "smithy4sOrg" -> organization,
```

(haven't tested it)

Reply:
There was a little more to it than that. Fixed.

Reply:
thanksss

Reply:
interesting, I thought it'd like a setting key

Reply:
That sure looked like it should work. Just slapping in `organization` doesn't work either. I suspect there's something clever to do with sbt's type system, but didn't find it.

My core complaint about sbt (and maven) is that it's not self-consistent enough to learn patterns and gain speed from experience.